tests/test_pytorch/test_datamodule.py

```diff
@@ -31,12 +31,12 @@
 
 
 def test_hpu_datamodule():
-    data_module = HPUDataModule(num_workers=8, batch_size=32, shuffle=False, pin_memory=True)
+    data_module = HPUDataModule(num_workers=2, batch_size=32, shuffle=False, pin_memory=True)
```

Comment:
Do num workers need to be the same as nb devices?

Reply:
No. This for the parallel data pre-process workers.